openage/cvar/config_file.py

```diff
@@ -31,4 +30,5 @@ def load_config_file(path, loaded=None):
                 set_cvar(split[1], " ".join(split[2:]))
             elif split[0] == "load" and len(split) >= 2:
                 for sub_path in split[1:]:
-                    load_config_file(os.path.join(os.path.dirname(path), sub_path), loaded)
+                    new_path = os.path.join(os.path.dirname(path), sub_path)
+                    load_config_file(new_path, set_cvar, loaded)
```

Comment:
i'd design the inclusion relative to the file that specifies the inclusion.
in `lol.cfg` i can then write `load bla/stuff.cfg`, where `lol.cfg` and the `bla` directory are in the same path.